servicetalk-grpc-internal/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2019, 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  implementation project(":servicetalk-annotations")
+  implementation project(":servicetalk-concurrent-api")
+  implementation project(":servicetalk-http-api")
```

Comment:
> Using them as implementation here doesn't affect their publication in other modules

If it doesn't break things, doesn't mean that it's correct. It just happens that the other modules are not broken because they already have `http-api`.

> this module only needs them as implementation. ie. it doesn't export any types which are defined in either concurrent-api or http-api.

`DeadlineUtils.java` has types from `http-api` as part of the public api => `http-api` have to be defined as `api`.

Reply:
Right, I had forgot I added those methods.